Remove reference to gtk internal functions.  These functions are no longer
available in recent versions of gtk, and sawfish doesn't use them anyway.  
Bug 48439, patch from fn_x

13 Mar 2005 agriffis

--- gtk.defs.orig	2005-01-14 02:58:25.104563416 +0100
+++ gtk.defs	2005-01-14 02:58:53.158298600 +0100
@@ -1717,15 +1717,6 @@
   ((GtkWindow window)
    (string geometry)))
 
-;; XXX GtkWindowGroup
-
-;; XXX marked as internal
-(define-func _gtk_window_internal_set_focus
-  none
-  ((GtkWindow window)
-   (GtkWidget focus))
-  (scm-name "gtk-window-set-focus"))
-
 ;;; HandleBox
 
 (define-object GtkHandleBox (GtkBin))
